inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/iceberg/IcebergSinkOperator.java

```diff
@@ -83,6 +87,15 @@ public StreamSink getFromEntity(StreamSinkEntity entity) {
         }
 
         IcebergSinkDTO dto = IcebergSinkDTO.getFromJson(entity.getExtParams());
+        if (StringUtils.isBlank(dto.getCatalogUri()) && dto.getCatalogType().equals(CATALOG_TYPE_HIVE)) {
```

Comment:
Suggest changing `dto.getCatalogType().equals(CATALOG_TYPE_HIVE)` to `CATALOG_TYPE_HIVE.equals(dto.getCatalogType())`.